doc/TYPES2021/Demos/HSP.tex

```diff
@@ -7,28 +7,30 @@ \section{Introduction}
 
 The first major milestone of the \agdaalgebras project is a proof of \emph{Birkhoff's
 variety theorem} (also known as the \emph{HSP theorem})~\cite{Birkhoff:1935}.
-To the best of our knowledge, this constitutes the first machine-verified proof of Birkoff's
-celebrated 1935 result.
+To the best of our knowledge, this constitutes the first ever formal proof of
```

Comment:
I think the 'ever' is a bit over the top?

Reply:
Agreed!